osbenchmark/client.py

```diff
@@ -210,6 +212,24 @@ def __init__(self, hosts, client_options):
         if self._is_set(self.client_options, "enable_cleanup_closed"):
             self.client_options["enable_cleanup_closed"] = convert.to_bool(self.client_options.pop("enable_cleanup_closed"))
 
+    @staticmethod
+    def _has_only_hostnames(hosts):
+        logger = logging.getLogger(__name__)
+        has_ip, has_hostname = False, False
+        for host in hosts:
+            if is_ipaddress(host["host"]):
+                has_ip = True
+            else:
+                has_hostname = True
+
+        if has_ip and has_hostname:
+            logger.warning("Although certificate verification is enabled, "
+                "peer hostnames will not be matched since the host list is a mix "
+                "of names and IP addresses")
```

Comment:
Please use:
```
console.warn("...", logger=self.logger)
```
so that the user gets feedback right away, rather than needing to look at the log file.

Reply:
Done1